### PR TITLE
Create Application of Box-Model in CSS.md

### LINKED ISSUE #6941: Application of Box-Model in CSS
### Description

Description
[Frontend]: Application of Box-Model in CSS

### Domain

Frontend Dev HTML/CSS/JS

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by varnikarathee**
<hr>

## Description 📜Application of Box-Model in CSS

Fixes #6941 
<hr>

## Type of change 📝Application of Box-Model in CSS

- [X ] Doc#umentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [X] Frontend Dev #HTML/CSS/JS

<hr>

## Checklist ✅

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [X] I have performed a self-review of my own code or work.
- [X] I have commented my code, particularly in hard-to-understand areas.
- [X] My changes generates no new warnings.
- [X] I'm GWOC'21 contributor

<hr>

### CODE REVIEW

**Review by 2024-SANDHYA — CHANGES_REQUESTED** [review]
Image credit & external links missing

File: Web_Development/FrontEnd/CSS_BoxModel/Application of Box-Model in CSS.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+3. BORDER:
+A property by which the user specifies and highlight the boundary of an object in the webpage. It helps an object appear more prominent and defined.
+
+4. MARGIN:
+A property by which the user adds up space outside the borders, to distingusih one object from another on a web page.
+
+### THE BLOCK DIAGRAM:   
+The block diagram shown as follows , highlights the different sections into which an entity is subdivided.      
+
+![image](https://user-images.githubusercontent.com/77229404/140058998-66e64981-6773-4215-82d6-e446a07fa4cd.png)
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/CSS_BoxModel/Application of Box-Model in CSS.md:22]: Add reference source for image.

File: Web_Development/FrontEnd/CSS_BoxModel/Application of Box-Model in CSS.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+<body>
+    <p class= "text">
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Accusamus impedit numquam corrupti iure, minus consequuntur voluptatibus quos laboriosam, adipisci tempora eos molestiae, consequatur in vel illo. Eaque quaerat odio repudiandae veritatis laudantium repellat deleniti exercitationem tempore consequatur quae beatae, ipsa iste aspernatur sint obcaecati officiis impedit! Aperiam libero deserunt ratione?
+    </p>
+</body>
+</html>
+```
+### THE OUTPUT: 
+The webpage would display following results:      
+![image](https://user-images.githubusercontent.com/77229404/140059134-786cc5bf-4a9d-407f-bdf5-b9a3f574ced7.png)
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/CSS_BoxModel/Application of Box-Model in CSS.md:52]: Add reference source for the image.

File: Web_Development/FrontEnd/CSS_BoxModel/Application of Box-Model in CSS.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+### THE OUTPUT: 
+The webpage would display following results:      
+![image](https://user-images.githubusercontent.com/77229404/140059134-786cc5bf-4a9d-407f-bdf5-b9a3f574ced7.png)    
+
+Here we observe that the content has been enclosed within a black border of 25 pixels.
+ 
+### THE BIFIRCATION:    
+For depiction of margin and padding we subdivide the given entity , via box method:     
+
+![image](https://user-images.githubusercontent.com/77229404/140059207-32f84ca7-99c2-4495-bd2c-436f4f6dbc72.png)
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/CSS_BoxModel/Application of Box-Model in CSS.md:59]: Add reference source for the image.

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]

**Comment by varnikarathee** [comment]
@kashishahuja2002 I had a doubt, here actually all the images are created by me in paint, then what will be the reference source in that case?

**Comment by 2024-SANDHYA** [comment]
> 
> 
> @kashishahuja2002 I had a doubt, here actually all the images are created by me in paint, then what will be the reference source in that case?

Then no need to mention the image credits, Just mention the reference links.

**Comment by varnikarathee** [comment]
@2024-SANDHYA actually the whole content is written by me . I haven't copied it from any source . Do i still need to mention any reference link ?

**Comment by kashishahuja2002** [comment]
> @2024-SANDHYA actually the whole content is written by me . I haven't copied it from any source . Do i still need to mention any reference link ?

Taking reference doesn't mean coping. You might have studied it from somewhere, and put it in your own words. So, you need to mention that link here.

**Comment by varnikarathee** [comment]
@kashishahuja2002 I have added external reference links. Kindly review.

**Review by kashishahuja2002 — APPROVED** [review]
LGTM

**Comment by varnikarathee** [comment]
@2024-SANDHYA Please merge my pr if all seems good to the team.

**Review by 2024-SANDHYA — APPROVED** [review]

**Pull request merged**